apps/public/.env.docker

```
@@ -2,4 +2,10 @@ DATABASE_HOST=mysql
 DATABASE_PORT=3306
 DATABASE_USER=root
 DATABASE_PASSWORD=CHANGE_ME
-DATABASE_NAME=mpga-data
\ No newline at end of file
+DATABASE_NAME=mpga-data
+
+REVALIDATE_SECRET=your-secret-here
+
+MATCH_PLAY_DEADLINE="2026-03-15"
+
+SEASON_YEAR=2025
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)